### PR TITLE
Fixed SonarCloud vulnerability

### LINKED ISSUE #740: ConfigKey.java: Make DEFAULT_PREFIX private and final
### Description
The following statement is reported as a vulnerability by [SonarCloud](https://sonarcloud.io/project/issues?id=jkubeio_jkube&open=AW3z0M4hL3SSO-VMZS4E&resolved=false&types=VULNERABILITY):

https://github.com/eclipse/jkube/blob/bf88230a5c110d686ea02ffb45dba8a294f5e2e9/jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/config/handler/property/ConfigKey.java#L153

should be changed to:
```java
    private static final String DEFAULT_PREFIX = "docker";
```

### Bonus

Create a unit test to verify your changes (intermediate).

#### Kubernetes

If you don't have a real Kubernetes cluster available (most probably), you can use Minikube or Kind to test with a local cluster.

- Minikube [get started guide](https://minikube.sigs.k8s.io/docs/start/)
- Kind (Requires Docker) [quick start](https://kind.sigs.k8s.io/docs/user/quick-start/)

#### OpenShift

If you don't have a real OpenShift cluster available (most probably), you can use [Red Hat's developer Sandbox for Red Hat OpenShift](https://developers.redhat.com/developer-sandbox). The only requirement is to have a Red Hat account.

Once you have your Sandbox environment, you'll need to download the oc tool from the cluster console.
(Press the `?` icon and from the context menu select `Command line tools`, you'll be redirected to https://$subdomain.openshiftapps.com/command-lines-tools where you'll be able to download the CLI for your platform)

### PR DESCRIPTION
**Pull request by param-thakker**
## Description
Fix #740 

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Comment by param-thakker** [comment]
Hi @manusa. I created a new pull request with the changes. I think it should be ok now! Please let me know if it works. This is my first OS contribution, so I'm trying my best.

**Comment by manusa** [comment]
Looking better now :+1:

**Review by manusa — COMMENTED** [review]
LGTM, thx

File: pom.xml [hunk 1]
```diff
@@ -86,7 +86,7 @@
     <version.maven-compiler-plugin>3.8.1</version.maven-compiler-plugin>
     <version.maven-enforcer-plugin>1.4.1</version.maven-enforcer-plugin>
     <version.maven-failsafe-plugin>3.0.0-M5</version.maven-failsafe-plugin>
-    <version.maven-gpg-plugin>1.6</version.maven-gpg-plugin>
+    <version.maven-gpg-plugin>3.0.1</version.maven-gpg-plugin>
```
Comment (rohanKanojia) [inline pom.xml:89]: Why is this change here?

Comment (param-thakker) [reply pom.xml:89]: @rohanKanojia it automatically made that change. I didn't edit this file at all!

Comment (rohanKanojia) [reply pom.xml:89]: Could you please revert this change and keep changes limited to issue scope?

**Review by rohanKanojia — APPROVED** [review]

**Comment by rohanKanojia** [comment]
Please squash your commits into one. ECA check is failing again due to signedoff footer not present in latest commit :-(

**Comment by param-thakker** [comment]
@rohanKanojia Is there anything else for me to do?

**Comment by rohanKanojia** [comment]
You still have two commits in your PR, but I guess we'll fix this by squash-and-merge option. LGTM from my side :heavy_check_mark:

**Review by manusa — APPROVED** [review]
LGTM, thx!

**Comment by param-thakker** [comment]
@manusa @rohanKanojia Thank you so much for your guidance and help throughout! This was my first ever contribution to OSS and I feel so good to have my changes merged to master branch. I would love to work on more interesting issues from here on! Please star my profile if possible. Also, let me know if I can work on anything else?

**Comment by rohanKanojia** [comment]
what do you mean by "Please star my profile if possible."?

**Comment by manusa** [comment]
I don't know if profiles can be "starred", but surely project can be ;)
